public/js/pickup_delivery.js

```diff
@@ -116,14 +116,15 @@ $(document).ready(function() {
   customers = [];
 
   var buildVRP = function() {
+    var correspondant = { 0: 'path_cheapest_arc', 1: 'global_cheapest_arc', 2: 'local_cheapest_insertion', 3: 'savings', 4: 'parallel_cheapest_insertion', 5: 'first_unbound', 6: 'christofides' }
     if (data.customers.length > 0 && data.vehicles.length > 0) {
       if (debug) console.log('Build json from csv: ', data);
       var vrp = {points: [], units: [], shipments: [], services: [], vehicles: [], configuration: {
         preprocessing: {
-          cluster_threshold: 0
+          cluster_threshold: 0,
+          first_solution_strategy: correspondant[parseInt($('#optim-solver-parameter').val())]
         },
         resolution: {
-          solver_parameter: parseInt($('#optim-solver-parameter').val()),
```

Comment:
Can't we directly display the heuristic names ?

Reply:
You mean, for customers ? That is an option but do they really know which heuristic corresponds to which number ?

Reply:
`solver_parameter` is deprecated. Every user should now use first_solution_strategy. We should not keep this logic even in facade.

It seems, to me, easier to associate resolution behavior to a name than to a number.

Reply:
As discussed today in daily point : we agreed that we need both number and name not to get our users confused because they don't know the correspondance yet.

Reply:
is this resolved or there are more things to do?

Reply:
It is okay by me because modification we discussed would take too long : changing JS file to output each number and heuristic name. I am not sure much customers use this front anyway ?

Reply:
In that case, @braktar can you take another look so that we can move this PR forward